### PR TITLE
fix `import attr` to use `import attrs`

### PR DESCRIPTION
**Pull request by chebbyChefNEQ**
Thanks to #508, I used `attr` instead of the correct package `attrs`

s/attr/attrs

### CODE REVIEW

File: python/lancedb/remote/__init__.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 import abc
 from typing import List, Optional
 
-import attr
```
Comment (eddyxu) [inline python/lancedb/remote/__init__.py:17]: Do you need to update `pyproject.toml`?

Comment (chebbyChefNEQ) [reply python/lancedb/remote/__init__.py:17]: Already done in #508

Comment (chebbyChefNEQ) [reply python/lancedb/remote/__init__.py:17]: `attr` and `attrs` are functionality the same

**Review by rec — APPROVED** [review]
Such service!

This change is not strictly necessary as `attrs` imports itself as `attr`, but I think it's a very good idea to prevent confusion.

  👍

**Review by changhiskhan — APPROVED** [review]
Thanks 🙏

**Pull request merged**